testlink/README.md

```diff
@@ -0,0 +1,24 @@
+testlink: Test Computer Science Ontology
+=======
+This ontology tests concepts, features, attributes for Computer Sciences.
+
+# Homepages
+* https://csolink.github.io/testlink-model/ -- testlink model
+* https://biolink.github.io/biolinkml/ -- Biolink modeling language
+
+# Docs
+* https://csolink.github.io/testlink-model/docs
+* https://biolink.github.io/biolinkml/docs
+
+# Vocabulary Usage
+    @prefix testlink: <https://w3id.org/testlink/v0.0.1#>
+
+# Contact
+* noel McLoughlin @gmail.com
+
+# Status
+* EXPERIMENTAL
```

Comment:
It may be better to omit the status here (and in csolink/README.md).  I assume over time it's likely to be more up-to-date in the target docs and easier to update there.

Reply:
Good idea, done!!